ui/app/styles/components/empty-state.scss

```diff
@@ -0,0 +1,37 @@
+.empty-state {
+  align-items: center;
+  color: $grey;
+  display: flex;
+  background: $ui-gray-050;
+  justify-content: center;
+  padding: $spacing-xxl $spacing-s;
+}
+
+.empty-state-content {
+  max-width: 320px;
+}
+
+.empty-state-title {
+  color: $grey;
+  font-size: $size-4;
+  font-weight: $font-weight-semibold;
+  line-height: 1.2;
+  margin-bottom: $spacing-xs;
+}
+
+.empty-state-actions {
+  margin-top: $spacing-xs;
+
+  a,
+  .link,
+  a:not(.button):not(.file-delete-button):not(.tag) {
+    color: $blue;
```

Comment:
Do we have these styles in a var or mixin somewhere? I feel like we use them in a few places.

Reply:
No we don't, but that's a good idea